Java/src/main/java/Airport.java

```diff
@@ -13,107 +15,109 @@
 public class Airport {
     private List<? extends Plane> planes;
 
+    public List<PassengerPlane> getPassengerPlane() {
+        List<? extends Plane> planesList = this.planes;
+        List<PassengerPlane> passangerPlaneList = new ArrayList<>();
 
+        for (Plane plane : planesList) {
+            if (plane instanceof PassengerPlane) {
+                passangerPlaneList.add((PassengerPlane) plane);
+            }
+        }
 
-    public List<PassengerPlane> getPasPl() {
-        List<? extends Plane> l = this.planes;
-        List<PassengerPlane> x = new ArrayList<>();
-        for (Plane p : l) {if (p instanceof PassengerPlane) {x.add((PassengerPlane) p);}}
-        return x;
+        return passangerPlaneList;
     }
 
     public List<MilitaryPlane> getMilitaryPlanes() {
         List<MilitaryPlane> militaryPlanes = new ArrayList<>();
+
         for (Plane plane : planes) {
             if (plane instanceof MilitaryPlane) {
                 militaryPlanes.add((MilitaryPlane) plane);
-            } //if
-            else {
+            }
+        }
 
-            } // else
-        } //for
         return militaryPlanes;
     }
 
     public PassengerPlane getPassengerPlaneWithMaxPassengersCapacity() {
-        List<PassengerPlane> passengerPlanes = getPasPl();
+        List<PassengerPlane> passengerPlanes = getPassengerPlane();
         PassengerPlane planeWithMaxCapacity = passengerPlanes.get(0);
-        for (int i = 0; i < passengerPlanes.size(); i++) {
+
+        for (int i = 0; i < passengerPlanes.size() ; i++) {
```

Comment:
you can use stream to have a simple logic inside the method and at the same time do the filtering part